rs/cli/src/main.rs

```diff
@@ -0,0 +1,43 @@
+use clap::Parser;
+use sails_cli::program::ProgramGenerator;
+
+#[derive(Parser)]
+enum Commands {
+    #[command(name = "new-program")]
```

Comment:
`new` or `init` shorter

Reply:
I've been bearing in mind something like `new-client` so they will be distinguishable